introduction.md

```diff
@@ -1,3 +1,7 @@
 this is my introduction.
 
 JavaScript yo!
+
+* Git
+* JavaScript
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?